### PR TITLE
#270 use the new release process

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang@uk.ibm.com>

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
@johnament @OndrejM please review so that I can start the release process shortly. Thanks

File: pom.xml [hunk 1]
```diff
@@ -311,17 +308,48 @@
...
-            <id>eclipse-jarsigner</id>
-            <!-- turns on signing of JARs by the Eclipse signer plugin -->
-        </profile>
-        <profile>
-            <id>gpg-sign</id>
-            <!-- signs all artifacts, required for deploying to Maven Central -->
+            <id>release</id>
             <build>
                 <plugins>
+                    <plugin>
```
Comment (johnament) [inline pom.xml]: This plugin should not be present.

File: pom.xml [hunk 2]
```diff
@@ -74,24 +73,21 @@
 
     <scm>
         <connection>scm:git:https://github.com/eclipse/microprofile-config.git</connection>
-        <developerConnection>scm:git:https://github.com/eclipse/microprofile-config.git</developerConnection>
-        <url>https://github.com/eclipse/microprofile-config</url>
+        <developerConnection>scm:git:git@github.com:eclipse/microprofile-config.git</developerConnection>
```
Comment (johnament) [inline pom.xml]: We should leave the URL option present.

File: pom.xml (hunk 1 above)
Comment (Emily-Jiang) [reply pom.xml]: ok. removed.

File: spec/pom.xml [hunk 3]
```diff
@@ -60,7 +62,11 @@
                         </goals>
                         <configuration>
                             <backend>pdf</backend>
-                            <outputFile>${project.build.finalName}.pdf</outputFile>
+                            <attributes>
```
Comment (johnament) [inline spec/pom.xml]: You'll need to duplicate this section for the HTML output as well.

Comment (Emily-Jiang) [reply spec/pom.xml]: done. thanks. I copied what @pilhuhn has done for metrics. @pilhuhn you missed this part in the metrics spec/pom.xml as well.

**Comment by Emily-Jiang** [comment]
@johnament can you double check again to see whether you can approve the changes?

**Pull request merged**